src/pages/policies/policy-list.tsx

```diff
@@ -11,6 +11,9 @@ interface Props {
 }
 
 export default function PolicyList(props: Props) {
+  const policyIds = useStableIds(props.policies, (p) => p.policyId);
```

Comment:
If I understand this correctly, we have this here to prevent unnecessary re-rerenders?

Reply:
To prevent unnecessary  loads ja. We trigger a load for the remaining coverage when the `policyIds` changes, so using this `useStableIds()` hook gives me a stable reference to those ids so that we don't trigger unnecessary loads (the reference to `props.policies` changes often as we refresh the data)